trompace/mutations/controlaction.py

```diff
@@ -90,3 +90,50 @@ def mutation_modify_controlaction(controlaction_id: str, actionstatus: ActionSta
         args["error"] = error
 
     return mutation_create(args, UPDATE_CONTROLACTION)
+
+
+def mutation_add_controlaction_propertyvaluespecification(controlaction_id: str, propertyvaluespecification_id: str):
+    """Returns a mutation for adding a control action to a property value specification.
+    Arguments:
+        controlaction_id: The unique identifier of the control action.
+        propertyvaluespecification_id: The unique identifier of the property value specification.
+    Returns:
+        The string for the mutation foradding a control action to a property value specification.
+    """
+    return format_link_mutation("MergePropertyValueSpecificationPotentialAction", controlaction_id, propertyvaluespecification_id)
+    #return format_link_mutation("AddControlActionObject", controlaction_id, propertyvaluespecification_id)
+
+def mutation_add_controlaction_property(controlaction_id: str, property_id: str):
+    """Returns a mutation for adding a control action to a property value specification.
+    Arguments:
+        controlaction_id: The unique identifier of the control action.
+        property_id: The unique identifier of the property.
+    Returns:
+        The string for the mutation foradding a control action to a property.
+    """
+    return format_link_mutation("MergeControlActionAdditionalProperty", controlaction_id, property_id)
+    # return format_link_mutation("AddControlActionObject", controlaction_id, property_id)
+
+def mutation_add_controlaction_object(controlaction_id: str, object_id: str):
+    """Returns a mutation for adding a control action to an object, either a property value specification or a property
+    Arguments:
+        controlaction_id: The unique identifier of the control action.
+        object_id: The unique identifier of the object (property / property value specification).
+    Returns:
+        The string for the mutation for adding a control action to an object.
+    """
+    return format_link_mutation("AddControlActionObject", controlaction_id, object_id)
+
+def mutation_request_controlaction(controlaction_id: str, entrypoint_id:str, properties: list,
```

Comment:
in older versions of python, to specify a list you should import the `typing` module.

```py
from typing import List
properties: List[str]
```

Make sure that you add some documentation here, especially saying what the meaning of these fields are.